src/app/pages/data-protection/replication/replication-wizard/replication-wizard.component.ts

```diff
@@ -1027,7 +1027,7 @@ export class ReplicationWizardComponent {
     }
   }
 
-  parsePickerTime(picker: string) {
+  parsePickerTime(picker: string): { minute: string; hour: string; dom: string; month: string; dow: string } {
```

Comment:
This could be a localized interface at this point

Reply:
Fixed.